website/dbt-versions.js

```diff
@@ -0,0 +1,20 @@
+exports.versions = [
+  // {
+  //   version: "1.1",
+  //   EOLDate: "2023-01-01"
+  // },
+  {
+    version: "1.0",
+    EOLDate: "2023-05-20"
```

Comment:
@jtcohen6 how long do you want to have 1.0 show up in the docs?
```suggestion
    EOLDate: "2023-12-03"
```